<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="histogram"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>R0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="r0_box">
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_box">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <number>655777</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="a_box">
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="dispersion">
        <property name="text">
         <string>Дисперсия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sd">
        <property name="text">
         <string>СКО</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="expected">
        <property name="text">
         <string>Мат. ожидание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="check_equability">
        <property name="text">
         <string>Результат проверки равномерности:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lenght">
        <property name="text">
         <string>L</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="period">
        <property name="text">
         <string>P</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="enter">
      <property name="text">
       <string>enter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
